tests/periph_uart_nonblocking/main.c

```diff
@@ -19,14 +19,15 @@
  */
 
 #include <stdio.h>
-#include <xtimer.h>
+#include "ztimer.h"
+#include "periph_conf.h"
```

Comment:
Why do we need `periph_conf.h` now?

Reply:
To get the definition of `UART_TXBUF_SIZE`, the header was included in `xtimer.h`

Reply:
Ah all right, thanks for clarifying!